src/index.js

```diff
@@ -0,0 +1,141 @@
+  const bgValue = event.target.options[event.target.selectedIndex].value;
+  if( bgValue == 'sunny' ) {
+    document.body.style.backgroundImage = "url(/ada-project-docs/assets/sunshine.png)";      
+    } else if( bgValue == "cloudy" ){
+      document.body.style.backgroundImage = "url(/ada-project-docs/assets/cloudy.png)";      
+    } else if( bgValue == "rainy" ){
+      document.body.style.backgroundImage = "url(/ada-project-docs/assets/rain.png)";      
+    } else if( bgValue == "snowy" ){
+      document.body.style.backgroundImage = "url(/ada-project-docs/assets/snow.png)";      
+    }
+});
+
+const weatherEmojisandColor = () => {
+  let temp = state.temperature;
+  let color;
+  if (temp <= 49) {
+    curWeatherEmojis.textContent="❄️🥶❄️🥶";
+    document.body.style.backgroundImage = "url(/ada-project-docs/assets/snow.png)";
+    color="cornflowerblue";
+  } else if (temp <= 59) {
+    curWeatherEmojis.textContent="🧤🧣🧤🧣";
+    document.body.style.backgroundImage = "url(/ada-project-docs/assets/rain.png)";
+    color = "green";
+  } else if (temp <= 69) {
+    curWeatherEmojis.textContent="✅✅✅✅";
+    document.body.style.backgroundImage = "url(/ada-project-docs/assets/cloudy.png)";
+    color ="yellow";
+  } else if (temp <= 79) {
+    curWeatherEmojis.textContent="😅😎😅😎";
+    document.body.style.backgroundImage = "url(/ada-project-docs/assets/sunshine.png)";
+    color="orange";
+  } else {
+    curWeatherEmojis.textContent="🔥🔥🔥🔥";
+    document.body.style.backgroundImage = "url(/ada-project-docs/assets/dry.png)";
+    color="red";
+  }
+    let tempTag = document.getElementById("tempTag");
+    
+    tempTag.textContent = String(temp);
+    tempTag.className = color;   
+};
+
+const getLatLonAsync = async () => {
+  try {
+    const response = await axios.get("http://127.0.0.1:5000/location", {
+      params: {
+        q: state.city,
+      }});
+    state.latitude = response.data[0].lat;
+    state.longitude = response.data[0].lon;
+    getWeather();
+  } catch (error) {
+    console.log("City not found");
+  }};
+
+const getWeather = () => {
+  axios.get("http://127.0.0.1:5000/weather", {
+    params: {
+      lat: state.latitude,
+      lon: state.longitude,
+  }})
+  .then ((response) => {
+    const tempInF = Math.floor((response.data.main.temp - 273.15) * (9/5) + 32);
+    tempTag.textContent = tempInF
+    state.temperature = tempInF
+    weatherEmojisandColor();
+  })
+  .catch ((error) => {
+    console.log("error no temperature found");
+  })};
+
+const increaseTemp = () => {
+    state.temperature+=1;
+    weatherEmojisandColor();
+  };
+
+const decreaseTemp = () => {
+    state.temperature-=1;
+    weatherEmojisandColor();
+  };
+
+const resetCity = () => {
+  const defaultCity = document.getElementById("defaultCity");
+  defaultCity.value = "Portland, OR";
+  state.city = defaultCity.value;
+  getLatLonAsync();
+  weatherEmojisandColor();
+  nowShowingTemp();
+};
+
+const nowShowingTemp = () => {
```

Comment:
Consider renaming this method to use a verb, maybe something like `updateNowShowingTemp`